.github/ISSUE_TEMPLATE/bug-report--query-builder-.md

```diff
@@ -29,8 +29,17 @@ Run the `.toEdgeQL()` method on your query and print the result. Then copy the g
 
 Copy/paste the error, or drag/drop a screenshot displaying the error message. If there's no error, describe the expected behavior.
 
-
 **Versions (please complete the following information):**
- - OS: [e.g. macOS Ventura]
- - EdgeDB version [e.g. 2.0]
- - `edgedb-js` version [e.g. `0.20.10`]
+
+<!--
+For EdgeDB Version: run `edgedb query 'select sys::get_version_as_str()'` from your project directory (or run `select sys::get_version_as_str();` in the EdgeDB interactive shell).
+For EdgeDB CLI version: Run `edgedb --version` from anywhere
+For `edgedb-js` version: Run `npm list edgedb --depth=0` from your project directory
+For Node/Deno version: Run `node --version` or `deno --version`
+-->
+
+- OS (e.g. iOS):
+- EdgeDB version (e.g. `2.0`):
+- EdgeDB CLI version (e.g. `2.0`):
+- `edgedb-js` version (e.g. `0.20.10`;):
+- Node/Deno version:
```

Comment:
Can we also add `@edgedb/generate` version and typescript version.

Reply:
@jaclarke Added those two.